enterprise/frontend/src/embedding-sdk/components/private/PublicComponentStylesWrapper.tsx

```diff
@@ -48,3 +37,19 @@ export const PublicComponentStylesWrapper = styled.div`
     display: inline;
   }
 `;
+/**
+ * We can't apply a global css reset as it would leak into the host app but we
+ * can't also apply our entire css reset scoped to this container, as it would
+ * be of higher specificity than some of our styles.
+ *
+ * The reason why this works is two things combined:
+ * - `*:where(button)` doesn't increase specificity, so the resulting specificity is (0,1,0)
```

Comment:
I didn't want this to become a wall of text, but both * and :where don't increase specificity (something I learned in these past days)